saliency/metrics/pic_test.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Tests for pic.py.
+
+To run: "python -m saliency.metrics.pic_test" from the top-level directory.
+"""
+
+import numpy as np
+import os
+import unittest
+
+from . import pic
+from pathlib import Path
+from PIL import Image
+
+
+class PicTest(unittest.TestCase):
+
+  def setUp(self):
+    self.dir_path = os.path.dirname(os.path.realpath(__file__))
+    self.root_path = Path(self.dir_path).parent.parent.absolute()
+    with Image.open(os.path.join(self.root_path, 'doberman.png')) as pil_img:
+      self.img = np.array(pil_img)
+
+  def _test_predict_func(self, input_images_batch):
+    # For the test purpose, use entropy as the prediction, so images
+    # with higher entropy produce higher model predictions. It works only,
+    # when the input has only one image in the batch.
+    entropy = pic.estimate_image_entropy(input_images_batch[0])
+    return [entropy]
+
+  def test_create_blurred_image_full_mask(self):
```

Comment:
these create_blurred_image functions both seem like pretty trivial tests. what about some blur where the result is more or less known (e.g. a 3x3 square with a 1 in the corners or something

Reply:
Added a new test.